lib/fastlane/plugin/wpmreleasetoolkit/helper/git_helper.rb

```diff
@@ -49,7 +49,7 @@ def self.cut_release_branch(branch_name)
       # @param [String] tag_version The name of the tag to cut the hotfix from
       # @param [String] new_version The name of the new version, e.g. "1.2.3"
       #
-      def self.create_branch_for_hotfix(tag_version, new_version)
+      def self.cut_hotfix_branch(tag_version, new_version)
```

Comment:
Nitpick – given the previous change to name things by purpose rather than use case, I wonder if this would make more sense as `branch_from_tag` or something?

Also – it seems like `create_branch` might already be able to do this from a tag, so do we need it at all?

Reply:
This seems to be a leftover of a previous intermediate step during my refactor. Did you review that change commit-per-commit? Normally this is supposed to be gone and replaced by `create_branch` in a later commit in this PR… 🤔

Reply:
Yep can confirm that's the case, sorry!